lib/beaker-answers/answers.rb

```diff
@@ -55,32 +76,42 @@ def self.supported_versions
     # @param [Array<Beaker::Host>] hosts An array of host objects.
     # @param [Hash] options options for answer files
     # @option options [Symbol] :type Should be one of :upgrade or :install.
+    # @option type [Symbol] :format Should be one of :bash or :hiera
```

Comment:
D'oh, this should be `type` as well, depending on how our discussion below sorts out.